test/TestHeapOrderingInvariant.t.sol

```diff
@@ -48,8 +49,8 @@ contract TestHeapOrderingInvariant is Test {
         uint256 length = heap.length();
 
         for (uint256 i; i < length; ++i) {
-            assertTrue((i * 2 + 1 >= length || i * 2 + 1 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 2 + 1))); // prettier-ignore
-            assertTrue((i * 2 + 2 >= length || i * 2 + 2 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 2 + 2))); // prettier-ignore
+            assertTrue((i * 2 + 1 >= length || i * 2 + 1 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 2 + 1)));// forgefmt: disable-line
+            assertTrue((i * 2 + 2 >= length || i * 2 + 2 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 2 + 2)));// forgefmt: disable-line
```

Comment:
weird that forge fmt removes the spacing here

Reply:
It didn't work on this line because of forge-fmt disable but I can add it manually.

Reply:
yes it is a bit weird.

Reply:
Indeed. Feel free to open an issue/PR on foundry :-)

Reply:
yes I cannot add it manually too. I believed I can but It was a mistake.